user-registration-app/src/app/views/registration-form/registration-form.component.html

```diff
@@ -643,4 +643,6 @@ <h5 class="accomplish">What would you like to get out of HackPSU?</h5>
     <li><a href="#mlh-container-coc">MLH Data Sharing Policy</a></li>
     <li><a href="#additional-questions-container">Additional Questions</a></li>
   </ul>
+
+  <p> Any issues? Email us at <a href="mailto:technology@hackpsu.org"> <b>technology@hackpsu.org </b> </a> </p>
```

Comment:
This looks great, but I think just UI wise it would be better to have this aligned with the index that's above it. So essentially, the beginning of this is aligned with "General" and the end is aligned with the end of "travel reimbursements." I think you could do this and keep the font the same by having:
"Any issues?
Email us at tech..."

And then the additional work is just getting the front part aligned